src/main/java/dev/morling/onebrc/CalculateAverage_royvanrijn.java

```diff
@@ -58,210 +57,173 @@
 public class CalculateAverage_royvanrijn {
 
     private static final String FILE = "./measurements.txt";
-    // private static final String FILE = "./src/test/resources/samples/measurements-10000-unique-keys.txt";
-
-    static final class Measurement {
-        int min, max, count;
-        long sum;
-
-        public Measurement() {
-            this.min = 1000;
-            this.max = -1000;
-        }
-
-        public Measurement updateWith(int measurement) {
-            min = min(min, measurement);
-            max = max(max, measurement);
-            sum += measurement;
-            count++;
-            return this;
-        }
 
-        public Measurement updateWith(Measurement measurement) {
-            min = min(min, measurement.min);
-            max = max(max, measurement.max);
-            sum += measurement.sum;
-            count += measurement.count;
-            return this;
-        }
+    private static final Unsafe UNSAFE = initUnsafe();
+    private static final boolean isBigEndian = ByteOrder.nativeOrder().equals(ByteOrder.BIG_ENDIAN);
 
-        public String toString() {
-            return round(min) + "/" + round((1.0 * sum) / count) + "/" + round(max);
+    private static Unsafe initUnsafe() {
```

Comment:
I believe you can just export `jdk.internal.misc.Unsafe` to your application in the execution script and do `getUnsafe()`.